Modified description and release (#110)
nasa-gcn/gcn-schema: README.md

# GCN Schema

> This repository contains the schemas for crafting new GCN notices. It also includes mission-specific schemas and illustrative examples, which are distributed by [GCN Kafka](https://gcn.nasa.gov). The formulation of mission schemas are one step in the process for setting up [New Notice Producers](https://gcn.nasa.gov/docs/producers).

Reviewer: ```suggestion
This repository contains the schemas for crafting new GCN notices. It also includes mission-specific schemas and illustrative examples, which are distributed by [GCN Kafka](https://gcn.nasa.gov). The formulation of mission schemas is one step in the process for setting up [New Notice Producers](https://gcn.nasa.gov/docs/producers).
```

> This repository contains the schemas necessary for crafting new GCN notices. It also includes mission-specific schemas and illustrative examples, which are distributed by [GCN Kafka](https://gcn.nasa.gov). The formulation of mission schemas are one step in the process for setting up [New Notice Producers](https://gcn.nasa.gov/docs/producers).

Reviewer: "Necessary" is redundant.

```suggestion
This repository contains the schemas for crafting new GCN notices. It also includes mission-specific schemas and illustrative examples, which are distributed by [GCN Kafka](https://gcn.nasa.gov). The formulation of mission schemas are one step in the process for setting up [New Notice Producers](https://gcn.nasa.gov/docs/producers).
```

> This repository contains the schemas necessary for crafting new GCN notices. It also includes mission-specific schemas and illustrative examples, which are distributed by [GCN Kafka](https://gcn.nasa.gov). These schemas are one of the important step in the process of establishing [New Notice](https://gcn.nasa.gov/docs/producers) for both existing and new missions.

Reviewer: "These schema are one of the important step": grammatical error, number agreement. Also, this wording begs the question what the _other_ important steps are.

"process of establishing New Notice for both existing and new missions": "New" should not be capitalized. Also, it looks like there is a missing word here.

Author: ```suggestion
This repository contains the schemas necessary for crafting new GCN notices. It also includes mission-specific schemas and illustrative examples, which are distributed by [GCN Kafka](https://gcn.nasa.gov). The formulation of mission schemas are one step in the process for setting up [New Notice Producers](https://gcn.nasa.gov/docs/producers).
```

> This repository contains the schemas essential schemas necessary for crafting new GCN notices. It also includes mission-specific schemas and illustrative examples, which are distributed by [GCN Kafka](https://gcn.nasa.gov). These schemas are one of the important step in the process of establishing [New Notice](https://gcn.nasa.gov/docs/producers) for both existing and new missions.

Author: ```suggestion
This repository contains the schemas necessary for crafting new GCN notices. It also includes mission-specific schemas and illustrative examples, which are distributed by [GCN Kafka](https://gcn.nasa.gov). These schemas are one of the important step in the process of establishing [New Notice](https://gcn.nasa.gov/docs/producers) for both existing and new missions.
```

> This repository contains the schemas rthe essential schemas necessary for crafting new GCN notices. It also includes mission-specific schemas and illustrative examples, which are distributed by [GCN Kafka](https://gcn.nasa.gov). These schemas are one of the important step in the process of establishing [New Notice](https://gcn.nasa.gov/docs/producers) for both existing and new missions.

Author: ```suggestion
This repository contains the schemas essential schemas necessary for crafting new GCN notices. It also includes mission-specific schemas and illustrative examples, which are distributed by [GCN Kafka](https://gcn.nasa.gov). These schemas are one of the important step in the process of establishing [New Notice](https://gcn.nasa.gov/docs/producers) for both existing and new missions.
```

> The alert format for new Notices over GCN Kafka is JSON. We have designed a set of core schema which serve as the building blocks for new GCN Notices. Instrument-specific schema can also be created, but we request that you utilize the core schema as much as possible.

Author: ```suggestion
The alert format for new Notices over GCN Kafka is JavaScript Object Notation (JSON), a widely adopted internet data format. We have designed a set of core schema which serve as the building blocks for new GCN Notices. Instrument-specific schema can also be created, but we request that you utilize the core schema as much as possible.
```

> The alert format for new Notices over GCN Kafka is JavaScript Object Notation (JSON), a widely adopted internet data format. We have designed a set of core schema which serve as the building blocks for new GCN Notices. Instrument-specific schema can also be created, but we request that you utilize the core schema as much as possible.

Reviewer: ```suggestion
The alert format for new Notices over GCN Kafka is JavaScript Object Notation (JSON), a widely adopted Internet data format. We have designed a set of core schema which serve as the building blocks for new GCN Notices. Instrument-specific schema can also be created, but we request that you utilize the core schema as much as possible.
```

Please add your schema to this repository under <code>gcn/notices/<i>mission</i>/</code> and submit a pull request for the GCN Team to review. The GCN team is happy to iterate with the producers on their schema contents and format. Your pipeline will generate JSON files following these schema and send alerts to GCN as described in [New Notice Producers](https://gcn.nasa.gov/docs/producers).

## How To Release

1.  Clone the parent repo (nasa-gcn/gcn-schema).

        git clone git@github.com:nasa-gcn/gcn-schema.git
        cd gcn-schema

> 2.  Use the following command to initiate the [release](https://semver.org) process: `npm version [ major | minor | patch ]`

Reviewer: ```suggestion
2.  Tag a new version by running the following command: `npm version [ major | minor | patch ]`. Choose `major`, `minor`, or `patch` depending on the kind of update according to [Semantic Versioning](https://semver.org) rules.
```

    This command will handle the intermediate steps of updating and committing the path changes in each file as defined in the `version` and `postversion` npm scripts

3.  Review the changes with `git log -p` to make sure that each file is appropriately updated.

4.  Finally, push both the commit and the tag:

        git push && git push origin <tag name>

> This will make release officially available for use.

Reviewer: ```suggestion
This will make release available for use.
```
